openstack/nova/templates/update-cells-job.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if hasPrefix "queens" .Values.imageVersionNova }}
```

Comment:
I would suggests to do the opposite: `not hasPrefix  "mitaka"`, as we will probably update to rocky soonish, and still run this job.

Reply:
I will remove all prefixes / if statements after upgraded all regions. So it make no difference really